recipes/entt/3.x.x/conanfile.py

```diff
@@ -4,19 +4,17 @@
 from conans.errors import ConanInvalidConfiguration
 
 
-class ConanRecipe(ConanFile):
+class EnttConan(ConanFile):
     name = "entt"
-
     description = "Gaming meets modern C++ - a fast and reliable entity-component system (ECS) and much more"
     topics = ("conan," "entt", "gaming", "entity", "ecs")
-
     homepage = "https://github.com/skypjack/entt"
     url = "https://github.com/conan-io/conan-center-index"
-
     license = "MIT"
-
     settings = "os", "compiler", "build_type", "arch"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "cmake"
```

Comment:
This recipe has no own `CMakeLists.txt` and does not call `conan_basic_setup()` anywhere.

Reply:
ok, generators removed, but also most of settings values (except compiler), and added no_copy_source